lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1125,53 +1159,36 @@ export class UserAgentApplication {
    * @param {string} [hash=window.location.hash] - Hash fragment of Url.
    */
   private handleAuthenticationResponse(hash: string): void {
-    // retrieve the hash
-    if (hash == null) {
-      hash = window.location.hash;
-    }
+    const locationHash = hash || window.location.hash;
 
-    let self = null;
-    let isPopup: boolean = false;
-    let isWindowOpenerMsal = false;
+    let isPopupOrIframe = false;
```

Comment:
nit: consider rename let to `isTopWindow`? not clear to me what the boolean is indicating.

Reply:
`isPopupOrIframe` represents whether or not the flow is a hidden iframe or a popup, as opposed to redirect (which would be the top frame). I'll see if there's a better name.